pkgs/development/python-modules/marimo/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, setuptools
+, click
+, jedi
+, markdown
+, pymdown-extensions
+, pygments
+, tomlkit
+, uvicorn
+, starlette
+, websockets
+, docutils
+, black
+}:
+
+buildPythonPackage rec {
+  pname = "marimo";
+  version = "0.3.3";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-l5QehU/LqEWb7ybKxace4sm6C9mcNlaHNOp55ExglpQ=";
+  };
+
+  propagatedBuildInputs = [
+    setuptools
+    click
+    jedi
+    markdown
+    pymdown-extensions
+    pygments
+    tomlkit
+    uvicorn
+    starlette
+    websockets
+    docutils
+    black
+  ];
+
+  meta = with lib; {
+    description = "A reactive notebook for Python — run reproducible experiments, execute as a script, deploy as an app, and version with git.";
```

Comment:
I think a better description should be chosen for this.
I like the one in `README.md` the most:
"A reactive Python notebook that's reproducible, git-friendly, and deployable as scripts or apps"

Also, descriptions should not have a period at the end.